pkg/platform/model/recipe.go

```diff
@@ -151,9 +165,6 @@ func fetchRecipeID(commitID strfmt.UUID, owner, project, orgID string, private b
 	if err != nil {
 		return nil, FailOrderRecipes.Wrap(err)
 	}
-	if private {
```

Comment:
Why is that not needed anymore?  Can we then also remove the arguments `private` and `orgID`?

Reply:
They removed it from the inventory API. I'll follow up on the greater refactoring in another PR.